File: NUnit/Version3/UnitTests/RestAPI/FlowControl/FlowControl.cs
namespace UnitTests.RestAPI.FlowControl {
    public class FlowControl {
        public Remaining remaining { get; set; }
    }

    public class Remaining {
        public int overall { get; set; }
        public int mac { get; set; }
        public int manual { get; set; }
    }
}
/*
 * Copyright Andrew Gray, Full Circle Solutions
 * Date: 18th September 2014
 * 
 */

File: JUnit/saucery-core/src/main/java/au/com/fullcirclesolutions/saucery/restapi/flowcontrol/json/FlowControl.java
package au.com.fullcirclesolutions.saucery.restapi.flowcontrol.json;

import com.fasterxml.jackson.annotation.JsonProperty;

public class FlowControl {
    @JsonProperty("remaining")
    public Remaining remaining;
}
/*
 * Copyright: Andrew Gray, Full Circle Solutions
 * Date: 26th March 2014 
 */